# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../doxygen/gcc41.patch
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

diff -Nur doxygen-1.4.6-orig/qtools/qvaluestack.h doxygen-1.4.6/qtools/qvaluestack.h
--- doxygen-1.4.6-orig/qtools/qvaluestack.h	2003-12-14 22:00:57.000000000 +0000
+++ doxygen-1.4.6/qtools/qvaluestack.h	2006-03-30 12:57:52.241204000 +0000
@@ -54,7 +54,7 @@
     {
 	T elem( this->last() );
 	if ( !this->isEmpty() )
-	    remove( this->fromLast() );
+	    this->remove( this->fromLast() );
 	return elem;
     }
     T& top() { return this->last(); }
